message_definitions/v1.0/messages.xml

```diff
@@ -1961,6 +1978,9 @@
     </message>
 
     <message name="STAB_ATTITUDE" id="216">
```

Comment:
This message is too long for modems like XBee that as a limit around 100 bytes per frame if I'm not wrong. Since we are using a lot these modems, we won't be able to receive this message.

Reply:
That is a good point. Counterintuitively, the changes that I am proposing are making  the message smaller for me ( added 6 floats, removed 9 floats one per actuator). I do understand though that it might increase the size of the message for you. What about grouping similar entries (like referece signals on the three axis) into a single array? That way one can chose to not stream the full array but just a single "0". I think that this should reduce the message size.

Reply:
@gautierhattenberger what do you think of the changes to make the message lighter? see also (https://github.com/paparazzi/paparazzi/pull/3329/commits/5be693940a237333ad6760a7b35b3c6c60e37aee#diff-1f4c42b588602cc40fd877f3ef6021ca0c926030a5d5df4e73bd7cb935b7d23dR156)

Reply:
That can work if the default size of the full message is less than 100 bytes. It means that in the code of AP, some elements are not activated by default, such as jerk or command u.
As a side note, the "new" realtime plotter in python doesn't support the arrays, so it would be nice to fix that as well...